src/test/regress/expected/citus_shards.out

```diff
@@ -0,0 +1,37 @@
+CREATE SCHEMA citus_shards;
+SET search_path TO citus_shards;
+SET citus.shard_count TO 4;
+SET citus.shard_replication_factor TO 1;
+SET citus.next_shard_id TO 99456900;
+ALTER SEQUENCE pg_catalog.pg_dist_colocationid_seq RESTART 456900;
+CREATE TABLE t1 (i int);
+SELECT create_distributed_table('t1', 'i');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE "t with space" (i int);
+SELECT create_distributed_table('"t with space"', 'i');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO t1 SELECT generate_series(1, 100);
+INSERT INTO "t with space" SELECT generate_series(1, 1000);
+SELECT * FROM citus_shards;
+   table_name   | shardid  |              shard_name              | citus_table_type | colocation_id | nodename  | nodeport | shard_size
+---------------------------------------------------------------------
+ "t with space" | 99456904 | citus_shards."t with space_99456904" | distributed      |        456900 | localhost |    57637 |      40960
+ "t with space" | 99456905 | citus_shards."t with space_99456905" | distributed      |        456900 | localhost |    57638 |      40960
+ "t with space" | 99456906 | citus_shards."t with space_99456906" | distributed      |        456900 | localhost |    57637 |      40960
```

Comment:
maybe reduce the columns to avoid flakiness

Reply:
The values don't cause flakiness because I reset the relevant sequences at the top of this test to the expected values.  I'd rather keep all columns instead so we are actually testing that the the full view works.

Reply:
I was mainly thinking of sizes, those tend to have some variability, but maybe it's fine

Reply:
That's fair, we usually start using a range there when they turn out to be variable. I think in this case it's not a problem. But to be clear, the sizes were specifically the issue with the tables with spaces, so I definitely want to include them.